### PR TITLE
FIX SECURITY SCANS ON WORKER: As Dan, I want the worker to have the security scans to pass so that we are confident the app is secure.

### PR DESCRIPTION
**Pull request by Reinkaos**
**Acceptance Criteria**
- `bundle audit`, `yarn audit` and `brakeman` tests are green
- The CI jobs doesn't allow failure anymore

**Notes**

- Gem vulnerabilities:
rack

- Brakeman overview:
Controllers: 9
Models: 22
Templates: 3
Errors: 0
Security Warnings: 8

- Node packages: 0 vulnerabilities found 
- Spun out of https://www.pivotaltracker.com/story/show/169559944

### CODE REVIEW

File: app/controllers/link_check_rules_controller.rb [hunk 1]
```diff
@@ -36,6 +36,7 @@ def update
 
   private
     def link_check_rule_params
-      params.require(:link_check_rule).permit!
+      params.require(:link_check_rule).permit(:source_id, :xpath, :status_codes, :active, :throttle,
+                                              :collection_title, :_id, :created_at, :updated_at)
```
Comment (invke) [inline app/controllers/link_check_rules_controller.rb]: Did you want to be permitted `created_at` and `updated_at`? Just checking this was intentional — it looks like it is but I don't know the context 😄

Comment (Reinkaos) [reply app/controllers/link_check_rules_controller.rb]: Yeah it's kinda strange but we seem to send these params from the manager to the worker

File: Gemfile [hunk 2]
```diff
@@ -20,12 +20,15 @@ gem 'oai'
 gem 'ougai', '~> 1.8'
 gem 'parse-cron'
 gem 'puma', '~> 3.12'
-gem 'rails', '~> 5.2.3'
+gem 'rails', '~> 5.2.4'
 gem 'responders'
 gem 'sidekiq', '~> 5.2.3'
 gem 'sinatra', require: nil
-gem 'supplejack_common', github: 'DigitalNZ/supplejack_common', tag: 'v2.8.0'
+gem 'supplejack_common', github: 'DigitalNZ/supplejack_common', branch: 'rm/fix-secutity-scans-on-worker'
```
Comment (daveharris) [inline Gemfile]: When does this get merged and tagged?

Comment (Reinkaos) [reply Gemfile]: I'll merge and tag sj common when the story is accepted :)

**Pull request merged**